src/components/AdminDashboard/CompleteIncident.js

```diff
@@ -92,17 +90,17 @@ const CompleteIncident = props => {
 
             {/* Location */}
             <div className="dropdown-text-wrap">
-              <p className="complete-incident-dropdown-titles-bold">Location:</p>
+              <p className="complete-incident-dropdown-titles-bold">
+                Location:
+              </p>
               <p className="location-dropdown-wrap">
                 {incident.city}, {incident.state}
               </p>
             </div>
 
             {/* Title */}
             <div className="dropdown-text-wrap">
-              <p className="complete-incident-dropdown-titles-bold">
-                Title:
-              </p>
+              <p className="complete-incident-dropdown-titles-bold">Title:</p>
               <p>{incident.title || '(none)'}</p>
```

Comment:
Sure I can show you what I mean by inconsistencies, one example would be on lines 93-95 you separated the p element into three lines while on line 103 you collapsed a similar p element to one line. this happens a couple times.

Reply:
Ah, ok gotcha. Yeah, most of this file was already built out. All I really did was add in the two blocks labeled "Confidence Rating". I'm planning to go through in a separate PR to correct some of these inconsistencies that are found in several files and do some cleaning up with that. More than happy to include that into this PR though if you think that would be better. Thought?

Reply:
Honestly it's not a problem, the code still works the same. Do you dislike it? Is it hard for you to read? if not, it's probably good for now.